tests/Unit/Evolution/Systems/ScalarWave/TestFunctions.py

```diff
@@ -0,0 +1,63 @@
+# Distributed under the MIT License.
```

Comment:
I think we've been naming these files now `Blah.py` if the test cpp file is named `Test_Blah.cpp` so we have a python file for each test file.

Reply:
renamed